apps/E2E/package.json

```diff
@@ -45,34 +44,5 @@
     "@babel/core": "^7.8.0",
     "@babel/runtime": "^7.8.0",
     "@fluentui-react-native/focus-zone": "^0.11.14"
-  },
-  "rnx-kit": {
-    "reactNativeVersion": "0.68",
-    "kitType": "library",
-    "bundle": [
-      {
-        "id": "win32_specs",
-        "entryFile": "src/index.win32.ts",
-        "bundleOutput": "dist/win32_specs.bundle.js",
-        "sourcemapOutput": "dist/win32_specs.bundle.map",
-        "assetsDest": "dist",
-        "targets": [
-          "native"
-        ],
-        "detectCyclicDependencies": {
-          "throwOnError": true
-        },
-        "detectDuplicateDependencies": {
-          "throwOnError": false
-        },
-        "typescriptValidation": false
-      }
-    ],
-    "capabilities": [
-      "core",
-      "react",
-      "metro-config",
-      "babel-preset-react-native"
-    ]
```

Comment:
Is there a reason why we're removing the entire rnx-kit config here? I would expect you'd just want to remove the "bundle" portion.

Reply:
Initially, I thought that rnx-kit was strictly for bundling in this package, but looking back, I am wrong. Adding back in newest commit (without bundle portion). Thanks